src/commcare_cloud/fab/fabfile.py

```diff
@@ -311,17 +311,6 @@ def _setup_release(keep_days=2, full_cluster=True):
     print(blue(env.code_root))
 
 
-def conditionally_stop_pillows_and_celery_during_migrate():
-    """
-    Conditionally stops pillows and celery if any migrations exist
-    """
-    if all(execute(db.migrations_exist).values()):
-        execute_with_timing(supervisor.stop_pillows)
-        execute(db.set_in_progress_flag)
```

Comment:
Do we need to also change the place where this flag gets unset?

Reply:
It doesn't get unset. It sets a redis key with an automatic timeout of 5 minutes. This same management command is also run later before the final service restart. It prevents service checks from failing (by ignoring failures) during that 5-minute period.

https://github.com/dimagi/commcare-cloud/blob/bd5ee374de19108520829d074aa6580d740466a5/src/commcare_cloud/fab/fabfile.py#L475